docs/src/dev-guide/tooling-containers.md

```diff
@@ -0,0 +1,55 @@
+# Containers
+
+We publish (to [GitHub packages][gh-packages]) several Docker container images useful for building
+and running CLP:
+
+* An [image][1] containing the dependencies necessary to build CLP core in a Centos 7.4 x86
```

Comment:
Same here, can we change 1-6 to be descriptive? E.g. [image][centos-7.4-dependencies]